R/ard_continuous.R

```diff
@@ -190,34 +206,27 @@ ard_continuous <- function(data,
   df_nested
 }
 
+process_stat_labels <- function(stat_labels, statistics){
```

Comment:
Can you update this helper function to begin with a `.`?

Reply:
Done!